modular_skyrat/modules/blueshield/code/blueshield.dm

```diff
@@ -71,15 +72,14 @@
 	head = /obj/item/clothing/head/helmet/space/plasmaman/blueshield
 	uniform = /obj/item/clothing/under/plasmaman/blueshield
 
+/obj/item/ammo_casing/energy/laser/hellfire/bs
+	projectile_type = /obj/projectile/beam/laser/hellfire
+	e_cost = 83 //Lets it squeeze out a few more shots
+	select_name = "maim"
 
-/obj/item/storage/box/gunset/blueshield
-	name = "CFA 'Lynx' gunset"
-	w_class = WEIGHT_CLASS_NORMAL
-
-/obj/item/storage/box/gunset/blueshield/PopulateContents()
-	. = ..()
-	new /obj/item/gun/ballistic/automatic/cfa_lynx/no_mag(src)
-	new /obj/item/ammo_box/magazine/multi_sprite/cfa_lynx(src)
-	new /obj/item/ammo_box/magazine/multi_sprite/cfa_lynx(src)
-	new /obj/item/ammo_box/magazine/multi_sprite/cfa_lynx(src)
-	new /obj/item/ammo_box/magazine/multi_sprite/cfa_lynx(src)
+/obj/item/gun/energy/laser/hellgun/blueshield
+	name = "\improper Mk.V Hellfire Lasergun"
+	desc = "An energy-based laser carbine that fires concentrated beams of light which pass through glass and thin metal. This model has been purpose built by updating the old hellfire pattern for the NanoTrasen Executive Retention Protocol. It has <b><span style='color:blue'>NanoTrasen</span></b> etched into the grip. "
```

Comment:
```suggestion
	desc = "An energy-based laser carbine that fires concentrated beams of light which pass through glass and thin metal. This model has been purpose built by updating the old hellfire pattern for the NanoTrasen Executive Retention Protocol. It has <b><span style='color:blue'>NanoTrasen</span></b> etched into the grip. "
```
So, we have things for spans, namely in this case `span_bold()` and `span_blue()`, but I've no clue how to use two in one.

You'd change it to be, say, "It has [span_bold(span_blue("Nanotrasen"))] etched into the grip.", I think? I dont know if the double-span would work, I can test it personally

Reply:
I took the span directly from CFA guns

Reply:
> I took the span directly from CFA guns

I figured, should probably be replaced there too in time. But dont follow in others' mistakes.
Again, I'll test to see the proper way to format 2 spans in a row for this system, gimme a mo'

Reply:
Nevermind, span procs arent compatiable with `desc`. Mark this as done and forgive my foolishness